public/app/plugins/panel/pmm-update/components/ProgressModal/ProgressModal.styles.ts

```diff
@@ -0,0 +1,78 @@
+import { css } from '@emotion/css';
+
+export const modal = css`
+  background-clip: padding-box;
+  background: #161719;
+  box-shadow: 0 3px 7px rgba(0, 0, 0, 0.3);
+  display: flex;
+  flex-direction: column;
+  left: 0;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 750px;
+  outline: none;
+  position: fixed;
+  right: 0;
+  top: 10%;
+  width: 100%;
+  z-index: 1050;
+  padding: 28px;
+`;
+
+export const outputHeader = css`
+  display: flex;
+
+  span {
+    flex: 1;
+  }
+`;
+
+export const clipboardButton = css`
+  margin-right: 8px;
+`;
+
+export const output = css`
+  height: 200px;
+  margin-right: 0;
+  margin-top: 15px;
+  overflow-y: scroll;
+  width: 100%;
+
+  pre {
+    background-color: #292929;
```

Comment:
Do the colors have to be hardcoded here? Doesn't it have issues in light mode?

Reply:
Well spotted. I got rid of hard-coded colors and used the theme